app.js

```diff
@@ -1,7 +1,7 @@
 console.log("js is linked");
 
 $(document).ready(function() {
-  console.log("jquery is linked");
+  console.log("document is ready");
 
   // get current weather
     let weatherUrl = 'http://api.wunderground.com/api/562b8535169e745a/conditions/q/CA/San_Francisco.json';
```

Comment:
This should be indented out one tab, as well as all the code that follows it.

It should be very obvious from the indentation what code belongs to which functions. It is not clear right now, which is why I had to reformat your code to be able to see what was going on.